com.hamrick.VueScan.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.hamrick.VueScan</id>
+	<name>VueScan</name>
+	<project_license>LicenseRef-proprietary</project_license>
+	<developer_name>Hamrick Software</developer_name>
+	<summary>Scanning software for 5000+ scanners</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	
+	<description>
+		<p>VueScan is a scanning program that is compatible with over 5680 scanners - and is available for Windows, Mac OS X and Linux.</p>
+	</description>
+	
+	<screenshots>
+		<screenshot type="default">
+			<image type="source">https://raw.githubusercontent.com/FakeShemp/flathub/com.hamrick.VueScan/screenshot.png</image>
```

Comment:
When this is accepted you should change the URL here to the actual flathub repo.

Reply:
For sure :)